### PR TITLE
configure probes in spring boot conventions

### PR DESCRIPTION
**Pull request by vudayani-vmw**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
This change is targeted for TAP 1.7 only

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Configure Liveliness Readiness Startup Probes for Spring Boot Applications in Tanzu Application Platform
```
Comment (karinepires) [inline spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md:1]: Please add alpha to the title, something like
"Configure Liveliness Readiness Startup Probes for Spring Boot Applications (alpha feature)"
I think we can omit TAP

File: toc.hbs.md [hunk 2]
```diff
@@ -535,6 +535,7 @@
       - [Spring Boot conventions overview](spring-boot-conventions/about.hbs.md)
       - [Install Spring Boot conventions](spring-boot-conventions/install-spring-boot-conventions.hbs.md)
       - [Configure Spring Boot actuators](spring-boot-conventions/configuring-spring-boot-actuators.hbs.md)
+      - [Configuring Liveliness Readiness Startup Probes](spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md)
```
Comment (karinepires) [inline toc.hbs.md:538]: Maybe it is worth to add alpha in the TOC as well: 
`      - [Configuring Liveliness Readiness Startup Probes (alpha)](spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md)`

**Review by karinepires — CHANGES_REQUESTED** [review]

**Comment by RichardJJG** [comment]
@vudayani-vmw, I've put this back in draft mode to stop this from being merged accidentally before @karinepires's feedback has been addressed. Please take it out of draft mode when it's ready for merging.

File: spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md (hunk 1 above)
Comment (vudayani-vmw) [reply spring-boot-conventions/configuring-liveliness-readiness-startup-probes.hbs.md]: Added to the title

**Comment by RichardJJG** [comment]
@vudayani-vmw, every instance of "[liveliness](https://github.com/pivotal/docs-tap/pull/2896/files#diff-74a4601b0f98c42e0eb08dc4fd9926c5245cb36b40809fe9a5e5f961ebb8b93dR46)" in the code should actually be "liveness", right? The word "liveliness" should actually be "liveness" everywhere, right?

**Comment by RichardJJG** [comment]
@vudayani-vmw, [where do you mean, specifically, when you say "earlier"?](https://github.com/pivotal/docs-tap/pull/2896/files#diff-74a4601b0f98c42e0eb08dc4fd9926c5245cb36b40809fe9a5e5f961ebb8b93dR74)?

**Comment by vudayani-vmw** [comment]
> @vudayani-vmw, every instance of "[liveliness](https://github.com/pivotal/docs-tap/pull/2896/files#diff-74a4601b0f98c42e0eb08dc4fd9926c5245cb36b40809fe9a5e5f961ebb8b93dR46)" in the code should actually be "liveness", right? The word "liveliness" should actually be "liveness" everywhere, right?
@RichardJJG That's correct! The Kubernetes term is "liveness". But currently, our [values](https://github.com/pivotal/docs-tap/pull/2896/files#diff-74a4601b0f98c42e0eb08dc4fd9926c5245cb36b40809fe9a5e5f961ebb8b93dR84) refer to "liveliness". We will push a fix for it to align with the k8s convention and update the docs.

**Comment by vudayani-vmw** [comment]
> where do you mean, specifically, when you say "earlier"

This line can be removed as we are listing the schema just above it.

**Comment by vudayani-vmw** [comment]
@RichardJJG 
This [PR](https://github.com/pivotal/docs-tap/pull/2905) addresses the above two issues you mentioned.

**Comment by RichardJJG** [comment]
@vudayani-vmw, you changed the version number to 1.7.0 [here](https://github.com/pivotal/docs-tap/pull/2896/files#diff-55a1153723b7bb1e2975dd62442c0893d129ca71a25abe6d984f18cd34d40dc7R37), [here](https://github.com/pivotal/docs-tap/pull/2896/files#diff-55a1153723b7bb1e2975dd62442c0893d129ca71a25abe6d984f18cd34d40dc7R48), [here](https://github.com/pivotal/docs-tap/pull/2896/files#diff-55a1153723b7bb1e2975dd62442c0893d129ca71a25abe6d984f18cd34d40dc7R100) and [here](https://github.com/pivotal/docs-tap/pull/2896/files#diff-55a1153723b7bb1e2975dd62442c0893d129ca71a25abe6d984f18cd34d40dc7R117).

Should these numbers always match the corresponding TAP minor? (In other words, should these numbers be 1.8.0 in the future docs and 1.6.0 in the TAP v1.6 docs and so on?) I ask because, if so, I can replace these version numbers with a variable that's tied to the TAP version.

**Comment by vudayani-vmw** [comment]
@RichardJJG I changed it to be more readable for the user who is working with that particular release.

**Comment by RichardJJG** [comment]
> @RichardJJG I changed it to be more readable for the user who is working with that particular release.

So, you don't want me to use TAP version variables in those places I linked to?

**Comment by vudayani-vmw** [comment]
@RichardJJG Please go ahead and use the TAP version variables as you suggested in all places. It will be consistent. Thanks

**Pull request merged**